package.json

```diff
@@ -12,6 +12,9 @@
   "devDependencies": {
     "@changesets/cli": "^2.26.2",
     "@pnpm/make-dedicated-lockfile": "^0.5.0",
-    "husky": "^8.0.1"
```

Comment:
why you remove `husky`

Reply:
it's not removed, line 16 :) it's just got ordered differently after adding those missing packages

@ujvl speculated on discord about maybe actually removing it, but there is actually something in the pre-commit hook, so better leave it as it is.